js/stdlib/ts/ALGO_compiled.ts

```diff
@@ -144,6 +145,21 @@ export const T_Address: ALGO_Ty<CBR_Address> = {
   }
 };
 
+export const T_Contract: ALGO_Ty<CBR_Contract> = {
+  ...CBR.BT_Contract,
+  netSize: 8,
+  toNet: (bv: CBR_Contract): NV => {
```

Comment:
Can we reuse the fields from T_UInt rather than copy the code? (Either by literally making `toNet: T_UInt.toNet` or by calling that function?